Makefile.helpers

```diff
@@ -23,21 +23,22 @@ define assert-unset
   @[ -z "$($1)" ] || (echo "$(1) should not be defined in $(@)"; exit 1)
 endef
 
-default:: $(DEFAULT_HELP_TARGET)
+default:: help
 	@exit 0
 
-.PHONY : help
-## This help screen
+## Help screen
 help:
-	@$(SELF) help/generate MAKEFILE_LIST="Makefile $(BUILD_HARNESS_PATH)/Makefile.helpers"
+	@printf "Available targets:\n\n"
+	@$(SELF) -s help/generate | grep -E "\w($(HELP_FILTER))"
```

Comment:
So I agree having a `helm/generate` base on a filter is nice. However, for simple, non-namespaced makefile targets, this has the opposite affect since there's no way to just say include the local targets from the `./Makefile`. 

All 100+ terraform modules now output the "simple" `make help` which are the targets from just the `Makefile`. 

This was achieved by `help/generate MAKEFILE_LIST="Makefile $(BUILD_HARNESS_PATH)/Makefile.helpers"`

I think we should have `help/short` run this command. 

Now, I'm not sure which behavior to set to `DEFAULT_HELP_TARGET` so we don't introduce regressions or have to submit hundreds more PRs. This is what that looks like: https://github.com/cloudposse/terraform-aws-ses-lambda-forwarder/blob/create-readme-yaml/docs/targets.md

Reply:
Fixed